### PR TITLE
Support gzipped PSL response

### LINKED ISSUE #87: UnicodeDecodeError when parsing public suffix list
Getting this error, looks like it's being caused by a recent update to the public suffix list at https://github.com/publicsuffix/list/blob/master/public_suffix_list.dat

```
alexlatchford@Alexs-MacBook-Air a8_nlp_tools (master) $ tldextract --version
tldextract 1.7.4
alexlatchford@Alexs-MacBook-Air a8_nlp_tools (master) $ tldextract http://forums.bbc.co.uk
Traceback (most recent call last):
  File "/usr/local/bin/tldextract", line 9, in <module>
    load_entry_point('tldextract==1.7.4', 'console_scripts', 'tldextract')()
  File "/Library/Python/2.7/site-packages/tldextract/tldextract.py", line 470, in main
    print(' '.join(extract(i))) # pylint: disable=superfluous-parens
  File "/Library/Python/2.7/site-packages/tldextract/tldextract.py", line 343, in extract
    return TLD_EXTRACTOR(url)
  File "/Library/Python/2.7/site-packages/tldextract/tldextract.py", line 214, in __call__
    suffix_index = self._get_tld_extractor().suffix_index(translations)
  File "/Library/Python/2.7/site-packages/tldextract/tldextract.py", line 254, in _get_tld_extractor
    raw_suffix_list_data = fetch_file(self.suffix_list_urls)
  File "/Library/Python/2.7/site-packages/tldextract/tldextract.py", line 375, in fetch_file
    return _decode_utf8(text)
  File "/Library/Python/2.7/site-packages/tldextract/tldextract.py", line 389, in _decode_utf8
    return unicode(text, 'utf-8')
UnicodeDecodeError: 'utf8' codec can't decode byte 0x8b in position 1: invalid start byte
```

Can't seem to do any URLs at all at the moment because the cache isn't loaded :(

### PR DESCRIPTION
**Pull request by jnozsc**
Closes #87

### CODE REVIEW

File: tldextract/tldextract.py [hunk 1]
```diff
@@ -366,8 +369,13 @@ def fetch_file(urls):
 
     for url in urls:
         try:
-            conn = urlopen(url)
-            text = conn.read()
+            response = urlopen(url)
+            if response.info().get('Content-Encoding') == 'gzip':
```
Comment (john-kurkowski) [inline tldextract/tldextract.py]: `urllib` makes everything so piecemeal... One day, I'm gonna break down and have this library depend on `python-requests`. ;)

**Comment by john-kurkowski** [comment]
Thanks for this!

**Comment by jnozsc** [comment]
Would you mind I request a new pip release in the following week(s)?

**Comment by john-kurkowski** [comment]
Sure! Done in [1.7.5](https://github.com/john-kurkowski/tldextract/commit/60ce797ce8d1c4b8c2fc9cab3d7a70096412f904).

**Pull request merged**